RedHatInsights/insights-inventory-frontend
# feat(ESSNTL-3736): Add more scenarios to the group deletion modal

Implements https://issues.redhat.com/browse/ESSNTL-3736.

It is requested to distinguish between empty and non-empty groups. For the latter, users cannot delete them until they remove all hosts associated with them. According to that, the group deletion modal should render the warning message and provide no options to remove such groups.

## How to test

Navigate to /inventory/groups and try to find both empty and non-empty groups. You should be able to remove the empty groups. You shouldn't be able to remove groups that have some hosts.

## Screenshots

![image](https://github.com/RedHatInsights/insights-inventory-frontend/assets/31385370/fcdaf91b-8121-4c96-ace9-42827588e635)

![image](https://github.com/RedHatInsights/insights-inventory-frontend/assets/31385370/f4c6911d-3c93-42fa-bb4e-743cd817d3e9)
